cmd/sebak/cmd/wallet/payment.go

```diff
@@ -150,6 +153,7 @@ func init() {
 	PaymentCmd.Flags().BoolVar(&flagCreateAccount, "create", flagCreateAccount, "Whether or not the account should be created")
 	PaymentCmd.Flags().BoolVar(&flagFreeze, "freeze", flagFreeze, "When present, the payment is a frozen account creation. Imply --create.")
 	PaymentCmd.Flags().BoolVar(&flagDry, "dry-run", flagDry, "Print the transaction instead of sending it")
+	PaymentCmd.Flags().BoolVar(&flagUnfreezingRequest, "unfreeze-request", flagUnfreezingRequest, "Request unfreezing for frozen account")
```

Comment:
@Geod24 I think unfreeze-request is more descriptive to user.
Because user is doing request for unfreeze not unfreeze when user generate transaction.

Reply:
Okay, it's not such an important point, so let's leave it as is.